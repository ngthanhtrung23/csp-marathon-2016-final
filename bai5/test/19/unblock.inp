11..23
445523
00672.
8967AA
89....
89BBCC
