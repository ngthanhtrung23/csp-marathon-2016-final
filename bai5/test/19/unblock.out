22
1 R 2
6 D 1
7 D 1
0 R 2
8 U 1
9 U 1
B L 2
6 D 1
7 D 1
A L 2
2 D 2
3 D 3
5 R 2
4 R 2
8 U 2
9 U 2
A L 2
6 U 1
7 U 1
C L 2
2 D 1
0 R 2

